### ISSUE TITLE
[Suggestion]

### ISSUE THREAD

**Issue by Pixl64** [MeteorDevelopment/meteor-client#938]
**Describe the feature**
A keybind into the Item Highlight / Search bar in chest GUIs so it is easier to find items

**Screenshots**
eg. https://media.forgecdn.net/attachments/thumbnails/252/143/310/172/unsearched.png
      https://media.forgecdn.net/attachments/thumbnails/252/144/310/172/searched.png

**Comment by RoastedCheese** [comment]
This would be awesome

**Comment by seasnail8169** [comment]
afaik this can be done with other mods?

**Comment by ClemGlacee** [comment]
Yes it can but why implementing a module that is only partially usable and efficient?
